subcommands/common_config.go

```diff
@@ -226,16 +226,13 @@ func SetUpdatesConfig(opts *SetUpdatesConfigOptions, reportedTag string, reporte
 	if len(configuredTag) == 0 && len(reportedTag) > 0 {
 		configuredTag = reportedTag
 	}
-	if len(configuredApps) == 0 && reportedApps != nil {
-		configuredApps = strings.Join(reportedApps, ",")
-	}
 
 	changed := false
 	if opts.UpdateApps != "" && configuredApps != opts.UpdateApps {
 		if strings.TrimSpace(opts.UpdateApps) == "," {
 			opts.UpdateApps = ""
 		}
-		fmt.Printf("Changing apps from: [%s] -> [%s]\n", configuredApps, opts.UpdateApps)
+		fmt.Printf("Changing apps to [%s]\n", opts.UpdateApps)
```

Comment:
Our users kind of liked the `from xyz to abc` thing.

After this change is approach, I think we need to extend the output we provide, not shrink it.
One idea that came to my mind is this:
```suggestion
		fmt.Printf(`
Currently configured apps: [%s]
Apps reported by the device: [%s]
Setting apps to [%s]
`, configuredApps, reportedApps, opts.UpdateApps)
```

Obviously, it is not that simple - there needs to be an if somewhere, as the `reported by device` part is only applicable to the device config.

Reply:
We may say `Apps installed on the device` or `Apps reported as installed on the device`.

Reply:
Added for both tags and apps the new lines now. Let me know what you guys think.